universum/modules/vcs/github_vcs.py

```diff
@@ -17,7 +21,34 @@ def get_time():
     return datetime.datetime.utcnow().replace(microsecond=0).isoformat() + "Z"
 
 
-class GithubMainVcs(ReportObserver, git_vcs.GitMainVcs):
+class GithubToken(Module):
+
+    @staticmethod
+    def define_arguments(argument_parser):
+        parser = argument_parser.get_or_create_group("GitHub", "GitHub repository settings")
+
+        parser.add_argument("--github-integration", "-ght", dest="integration_id", metavar="GITHUB_INTEGRATION",
+                            help="GitHub application ID (see 'general')")
+        parser.add_argument("--github-installation", "-ghs", dest="installation_id", metavar="GITHUB_INSTALLATION",
+                            help="in-project installation ID (see 'integrations & services')"
```

Comment:
Are you sure "in-project" is correct term?
I think github uses "repository" in the same context.

Reply:
Not sure at all what I meant back then when writing this script. Anyway, will be changed when updating documentation. Should be retrieved from webhook payload.